lively.ide/world.js

```diff
@@ -449,8 +449,13 @@ export class LivelyWorld extends World {
     versionChecker.name = 'lively version checker';
     versionChecker.openInWorld();
     versionChecker.relayout();
-    await versionChecker.checkVersion();
-
+    try {
+      await promise.timeout(1000, versionChecker.checkVersion());
```

Comment:
This should probably respect the OFFLINE_MODE and was an oversight. I will take a look at this, but I think it makes more sense to do this after you are done here, what do you think? The `timeout` would still make sense in any case I believe.